generators/upgrade/index.js

```diff
@@ -224,7 +224,7 @@ module.exports = UpgradeGenerator.extend({
         updateJhipster: function() {
             this.log(chalk.yellow('Updating ' + GENERATOR_JHIPSTER + '. This might take some time...'));
             var done = this.async();
-            shelljs.exec('npm install ' + GENERATOR_JHIPSTER, {silent:true}, function (code, msg, err) {
+            shelljs.exec('npm install ' + GENERATOR_JHIPSTER + '@' + this.latestVersion, {silent:true}, function (code, msg, err) {
```

Comment:
yes u r right, now that yarn is an option ... the fact is I don't know yarn, I'll need a little time to build a docker image with yarn + upgrade sub generator dev ; do u prefer to create a new issue, or is it better to include yarn in this PR ?